hs_core/irods.py

```diff
@@ -156,27 +165,28 @@ def list_ticket(self, ticket):
                     value = line[1]
                     if key == 'collection name' or key == 'data collection':
                         output['full_path'] = value
-                        if self.resource_federation_path:
+                        if self.is_federated:
                             if __debug__:
                                 assert(value.startswith(self.resource_federation_path))
                             output['long_path'] = value[len(self.resource_federation_path):]
-                            output['fed_path'] = self.resource_federation_path
+                            output['home_path'] = self.resource_federation_path
                         else:
                             location = value.search(self.short_id)
```

Comment:
@alvacouch Are you sure ```search()``` would work here? ```value``` appears to be a string, and there does not appear to be a ```search``` method for string, do you mean to use ```find``` instead?